core/arch/x86/decode_table.c

```diff
@@ -1781,6 +1790,12 @@ const instr_info_t * const op_instr[] =
 #define Me TYPE_M, OPSZ_16_vex32_evex64
 #define Ue TYPE_V_MODRM, OPSZ_16_vex32_evex64
 
+/* MPX additions, own codes */
+#define TRq_dq TYPE_T_REG, OPSZ_8_dimode16
+#define TMq_dq TYPE_T_MODRM, OPSZ_8_dimode16
+#define ETy  TYPE_E, OPSZ_4_dimode8
```

Comment:
Use the existing codes: I see 'r' for OPSZ_4x8.

Reply:
ETy => Er.
MTd_q => Mr.